index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Lesson Learned - A.AFL</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header id="cabecalho">
+      <h1 id="titulo">Lesson Learned</h1>
+      <h2>Alan Albuquerque Ferreira Lopes</h2>
+      <nav>
+        <a href="#licoes aprendidas">Lições</a>
```

Comment:
Alan, pude perceber que esse link nao estava funcionando pois o id esta diferente da sua `ol` la embaixo!
```suggestion
        <a href="#licoes_aprendidas">Lições</a>
```